README.md

```diff
@@ -103,17 +121,14 @@ In order to use Docker Compose, you will need to have the following software ins
 
 4. Wait for Docker Compose to complete, and the `BattDB` database has been successfully deployed. The data for the database will be stored in the directory `assets/Docker/data`.
 
-## Migrate Database
+### Using migration Scripts on your server
```

Comment:
@SyXuan I am thinking of this as a deployment option too. So this should be Indented further and be the third deployment option. Thanks

Reply:
Thanks for pointing this out!